docs/_index.md

```diff
@@ -42,6 +42,10 @@ See the **advanced** installation instructions to use a preview or alternate ver
 [`pipx`](https://github.com/pypa/pipx) is used to install Python CLI applications globally while still isolating them in virtual environments.
 `pipx` will manage upgrades and uninstalls when used to install Poetry.
 
+{{% note %}}
+pipx is the installation approach which the user should choose to install multiple versions of pipx. 
```

Comment:
even after you fix the brain fart, it is not clear to me that this is an improvement.  

This question is on your mind and so you think that the docs should emphasise it; but 99.9% of readers do not want to install multiple versions of poetry, and the remaining 0.1% can read how to do so in the very next section anyway.

ie the danger here is that this makes the docs better specifically for you - but a little worse for almost everyone.

Reply:
You refer to 99.9% of user which use pipx as the main installation method? If so I agree. So a better place would be inside the official installer section as a specific subsection?

I can only imagine, but for many users which used like me the official install and reached my point, it might be unclear that pipx is the solution.

Reply:
I refer to the 99.9% of users who don't want to install more than one version of poetry.

pipx is already documented as the primary installation method, and that section already clearly describes its use to install more than one version of poetry.  

No matter what the docs say, some people just won't read them - so at some point you hit diminishing returns:
- people who didn't read the docs didn't read the bit that you've added either
- people who did read the docs already had the answer

Anyway I don't hold the commit bit, I'll leave it to someone who does to take a view on whether this is worthwhile

Reply:
Thanks for clarification.

So for the 0.1% of the users (me included) who does not install with pipx in the first place and used the official installer and red the documentation only for the official installer, the hint in advanced section might be helpful.

Speaking for myself this sentence would have been useful for me.

Reply:
IMO, the note makes sense at the updated location - at least for users who are too fixated on the official installer.